src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/core/RcaConf.java

```diff
@@ -84,6 +89,10 @@ public String getConfigFileLoc() {
     return configFileLoc;
   }
 
+  public long getLastModifiedTime() {
```

Comment:
Let's add a comment as this is the last modified time of the rca.conf file itself

Reply:
Makes sense. Will add.